Keas.Mvc/ClientApp/components/People/PeopleContainer.tsx

```diff
@@ -18,33 +24,262 @@ export default class PeopleContainer extends React.Component<{}, IState> {
+          {!personAction && 
+            this._renderTableView()
+          }
+          {personAction === "details" && !!detailPerson && !!detailPerson.person &&
+            this._renderDetailsView(detailPerson.person)
+          }
+        </div>
+      </div>
+    );
+  }
+
+  private _renderTableView = () => {
+    let filteredPeople = this.state.people;
+    if(this.state.tagFilters.length > 0)
+    {
+      filteredPeople = filteredPeople.filter(x => this._checkTagFilters(x.person, this.state.tagFilters));
+    }
     return (
       <div>
-        <h2>
-          Eventually this will display everyone in team {this.context.team.name}
-        </h2>
-        {memberList}
+      <SearchTags tags={this.state.tags} selected={this.state.tagFilters} onSelect={this._filterTags} disabled={false}/>
+      <PeopleTable
+        people={filteredPeople}
+        showDetails={this._openDetailsModal}
+        onEdit={this._openEditModal}
+        filtered={this.state.tableFilters}
+        updateFilters={this._updateTableFilters}
+      />
       </div>
+    ); 
+  }
+
+  private _renderDetailsView = (detailPerson: IPerson) => {
+    return(
+      <PersonDetails
+        selectedPerson={detailPerson}
+        tags={this.state.tags}
+        goBack={this._goBack}
+        assignedOrCreated={this._assetAssigned}
+        revokedOrDeleted={this._assetRevoked}
+      />
     );
   }
+
+  private _updateTableFilters = (filters: any[]) => {
+    this.setState({tableFilters: filters});
+  }
+
+  private _createAndMaybeAssignEquipment = async (
+    person: IPerson,
+    equipment: IPerson,
+    date: any
+  ) => {
+    // // call API to create a equipment, then assign it if there is a person to assign to
+    // // if we are creating a new equipment
+    // if (equipment.id === 0) {
+    //   equipment.teamId = this.context.team.id;
+    //   equipment = await this.context.fetch(`/api/${this.context.team.name}/equipment/create`, {
+    //     body: JSON.stringify(equipment),
+    //     method: "POST"
+    //   });
+    // }
+
+    // // if we know who to assign it to, do it now
+    // if (person) {
+    //   const assignUrl = `/api/${this.context.team.name}/equipment/assign?equipmentId=${equipment.id}&personId=${
+    //     person.id
+    //   }&date=${date}`;
+
+    //   equipment = await this.context.fetch(assignUrl, {
+    //     method: "POST"
+    //   });
+    // }
+
+    // const index = this.state.equipment.findIndex(x => x.id === equipment.id);
+    // if (index !== -1) {
+    //   // update already existing entry in equipment
+    //   const updateEquipment = [...this.state.equipment];
+    //   updateEquipment[index] = equipment;
+
+    //   this.setState({
+    //     ...this.state,
+    //     equipment: updateEquipment
+    //   });
+    // } else {
+    //   this.setState({
+    //     equipment: [...this.state.equipment, equipment]
+    //   });
+    // }
+  };
+
+  private _revokeEquipment = async (equipment: IPerson) => {
```

Comment:
@laholstege what's all these commented out methods about?  Is it because we are using the underlying containers to do all the work now and these are no longer needed?  If so I say just dump them if you don't need it any more.